galactic/distribution.yaml

```diff
@@ -919,6 +919,24 @@ repositories:
       url: https://github.com/ros-drivers/four_wheel_steering_msgs.git
       version: ros2
     status: maintained
+  gazebo_ros2_control:
+    doc:
+      type: git
+      url: https://github.com/ros-simulation/gazebo_ros2_control.git
+      version: master
+    release:
+      packages:
+      - gazebo_ros2_control
+      - gazebo_ros2_control_demos
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ros2-gbp/gazebo_ros2_control-release.git
+      version: 0.0.4-1
```

Comment:
This is fine for now, but I recommend increasing the minor number for the next Galactic release. It's unusual that the older ROS distro Foxy has version 0.0.5 while the newer distro Galactic has 0.0.4.